### PR TITLE
Add "Lava Puddles by smugler5"

### PR DESCRIPTION
**Pull request by Avyukt1**

### CODE REVIEW

**Comment by Avyukt1** [comment]
My map now works with no technical errors. I fixed them all. But I just want you all to test and review my map.

File: Lava-Puddles-Map/map.conf [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+rotation = z
+name = Lava Puddles
+author = smugler5
+h = 140
+team.1.color = red
+team.2.color = blue
+team.2.pos = (101,1,-102)
+team.1 = red
+licence = CC BY-SA
+treasures = shooter:shotgun, default:pick_steel, default:pick_steel, default:dirt 70, default:stone 60
```
Comment (Panquesito7) [inline Lava-Puddles-Map/map.conf:14]: Incorrect, this should be `shooter_guns:shooter`.
```suggestion
treasures = shooter_guns:shotgun, default:pick_steel, default:pick_steel, default:dirt 70, default:stone 60
```

File: Lava-Puddles-Map/map.conf [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+others = This map is full of puddles with lava. Also there are diamond hidden somewhere.
```
Comment (Panquesito7) [inline Lava-Puddles-Map/map.conf:15]:
```suggestion
others = This map is full of puddles with lava. Also there is diamond hidden somewhere.
```

File: Lava-Puddles-Map/README.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# Lava Puddles by smugler5
+I request the minetest ctf team to add this map to their server.
```
Comment (Panquesito7) [inline Lava-Puddles-Map/README.md:2]: I don't think this is needed.

File: Lava-Puddles-Map/map.conf [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+team.2 = blue
+rotation = z
+name = Lava Puddles
+author = smugler5
+h = 140
+team.1.color = red
+team.2.color = blue
+team.2.pos = (101,1,-102)
+team.1 = red
+licence = CC BY-SA
```
Comment (Panquesito7) [inline Lava-Puddles-Map/map.conf:13]: What is the license version? Is it CC BY-SA 4.0 or is it 3.0?

Comment (Panquesito7) [inline Lava-Puddles-Map/map.conf:13]:
```suggestion
license = CC BY-SA
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
The directory name should be `lava_puddles`.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
I believe naming your directory `lava_puddles` would be better.

**Comment by Avyukt1** [comment]
Okay,
Thanks!
I am making the changes.

File: Lava-Puddles-Map/map.conf (hunk 4 above)
Comment (Avyukt1) [reply Lava-Puddles-Map/map.conf]: Okay,
Thanks!
I am making the changes.

**Comment by Avyukt1** [comment]
It is showing an error while uploading the files.

**Comment by Avyukt1** [comment]
I have deleted the other files but I cannot add my files. It is showing error.

**Comment by Avyukt1** [comment]
I think I should initialize the repository again.

**Pull request closed without merging**